storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -277,15 +277,34 @@
          (value-stats stats SPOUT-FIELDS)
          {:type :spout}))
 
+(defn values-divided-by [pairs t]
+  (let [update-values (fn [m f & args]
+                        (into {} (for [[k v] m] [k (apply f v args)])))]
+    (update-values pairs / (double t))))
```

Comment:
I would prefer to see update-values be a regular function instead of defined each time value-divide-by is called.